environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -878,6 +898,220 @@ objects:
         - name: telemeter-server-tls
           secret:
             secretName: telemeter-server-shared
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  rules:
+  - nonResourceURLs:
+    - /metrics
+    verbs:
+    - get
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: prometheus-ams-${NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: prometheus-ams-${NAMESPACE}
+  subjects:
+  - kind: ServiceAccount
+    name: prometheus-ams
+    namespace: ${NAMESPACE}
+- apiVersion: v1
+  data:
+    nginx.conf: |
+      daemon off;
+      worker_processes 1;
+      error_log /dev/stderr;
+      pid /tmp/nginx.pid;
+
+      events {
+          worker_connections 1024;
+      }
+
+      http {
+        log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                          '$status $body_bytes_sent "$http_referer" '
+                          '"$http_user_agent" "$http_x_forwarded_for"';
+
+        upstream app {
+          server http://thanos-receive.${NAMESPACE}.svc.cluster.local:10902/api/v1/receive;
+          keepalive 64;
+        }
+
+        server {
+          listen *:8080;
+          server_name _;
+          access_log  /dev/stdout  main;
+          error_log /dev/stderr;
+
+          location / {
+            proxy_set_header THANOS-TENANT ${THANOS_TENANT};
```

Comment:
ditto

Reply:
I think we should hardcode this in here. We are probably going to hardcode the same ID in the Telemeter v1 forwarding.

Reply:
Could we share this as a jsonnet variable between them to avoid synchronization issue?

Reply:
There is no variable replacement for `${THANOS_TENANT}`, this is not defined as a parameter anywhere. It also doesn't really make sense for it to be a parameter: this configuration stack serves every possible thanos tenant. Really, we need to launch one remote-write-proxy per tenant, in which case the header should be be hardcoded for that tenant.

Reply:
I'll use `_id` then as we have it right now.

Reply:
this would need to be an id not `_id` right?

Reply:
Yes, as I recall that's the default. That's why I suggested that.